Halmap/View/SongInformation/SongInformationView.swift

```diff
@@ -8,23 +8,22 @@
 import SwiftUI
 
 struct SongInformationView: View {
-
+    
     @State var music: Music
     
     var body: some View {
         VStack(alignment: .leading) {
             
             SongHeaderView(music: $music)
-                .frame(width: UIScreen.main.bounds.width,
+                .frame(width: UIScreen.main.bounds.width + 3,
```

Comment:
```suggestion
                .frame(width: UIScreen.main.bounds.width ,
```

+3 부분은 top image에 `.scaleToFill`로 해결해서 수정해도 될 것 같아요!